### PR TITLE
[xdoctest][task cleanup 376~381] reformat example code with google style

### PR DESCRIPTION
**Pull request by yuchen202**
### PR types

Others

### PR changes

Others

### Description

清理遗留代码

修改如下文件的示例代码，使其通过 xdoctest 检查：
python/paddle/distribution/bernoulli.py		
python/paddle/distribution/categorical.py		
python/paddle/distribution/gumbel.py		
python/paddle/hapi/dynamic_flops.py		
python/paddle/hapi/model.py		
python/paddle/hapi/model_summary.py

Related links
https://github.com/PaddlePaddle/Paddle/issues/55629
https://github.com/PaddlePaddle/Paddle/pull/55295
@sunzhongkai588 @SigureMo

请评审！

### CODE REVIEW

File: python/paddle/distribution/bernoulli.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ class Bernoulli(exponential_family.ExponentialFamily):
 
         .. code-block:: python
 
+            >>> # doctest: +REQUIRES(env:DISTRIBUTED)
```
Comment (SigureMo) [inline python/paddle/distribution/bernoulli.py:75]: 376-381 是添加 SKIP reason，这些额外修改的原因是？

File: python/paddle/distribution/categorical.py [hunk 2]
```diff
@@ -64,7 +65,7 @@ class Categorical(distribution.Distribution):
             >>> cat = Categorical(x)
             >>> cat2 = Categorical(y)
 
-            >>> # doctest: +SKIP
+            >>> # doctest: +SKIP('`paddle.distribution` can not run in xdoctest')
```
Comment (SigureMo) [inline python/paddle/distribution/categorical.py:68]: 并不是说直接 `xxx can not run in xdoctest` 就可以了，这样和直接空着没啥区别

File: python/paddle/hapi/model.py [hunk 3]
```diff
@@ -1140,7 +1140,7 @@ class Model:
             ...
             ...     model = paddle.Model(net)
             ...     optim = paddle.optimizer.SGD(learning_rate=1e-3, parameters=model.parameters())
-            ...
+
```
Comment (SigureMo) [inline python/paddle/hapi/model.py:1143]: 为什么要删？

File: python/paddle/distribution/bernoulli.py (hunk 1 above)
Comment (yuchen202) [reply python/paddle/distribution/bernoulli.py:75]: 那我恢复？

File: python/paddle/distribution/categorical.py (hunk 2 above)
Comment (yuchen202) [reply python/paddle/distribution/categorical.py:68]: 咋说呢，其实我是照着
https://github.com/PaddlePaddle/Paddle/pull/57676
来做的，不知道可否进一步指导一下？

**Comment by luotao1** [comment]
close due to the following PR is merged:
- https://github.com/PaddlePaddle/Paddle/pull/58173

**Pull request closed without merging**